frontend/src/components/App/Home/index.tsx

```diff
@@ -160,30 +161,106 @@ function ClusterStatus({ error }: { error?: ApiError | null }) {
 
 export default function Home() {
   const history = useHistory();
-  const clusters = useClustersConf() || {};
+  const clustersConf = useClustersConf() || {};
+  const clusters = useMemo(() => {
+    return clustersConf;
+  }, [clustersConf]);
```

Comment:
I thought memo didn't compare objects. If that's the case, this should not do much.